### PR TITLE
Pyro Integration

### PR DESCRIPTION
**Pull request by ChecksumDev**

### CODE REVIEW

File: .gitignore
Comment (Erb3) [inline .gitignore:1]: This change looks unintentional

Comment (brawaru) [reply .gitignore:1]: Actually, it's a good idea to have `.env` files ignored by Git.

File: apps/app-frontend/src/components/ui/servers/ServerListing.vue [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      </div>
+    </div>
+  </RouterLink>
+</template>
+
+<script setup lang="ts">
+import { ref, computed, onMounted } from 'vue'
+import { ChevronRightIcon } from '@modrinth/assets'
+import type { StatusState } from '@/components/ui/servers/ServerInstallStatusPill.vue'
+// i fucking hate whoever set up typescript in this god forsaken project
```
Comment (brawaru) [inline apps/app-frontend/src/components/ui/servers/ServerListing.vue:49]: :HUH:

File: .gitignore
Comment (Erb3) [reply .gitignore:1]: I was confusing repos, we actually have .env examples here, so you're right. The comment feels off though

**Comment by ChecksumDev** [comment]
Closing and moving development to upstream pyro branch to reopen.

**Pull request closed without merging**